lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -474,7 +476,10 @@
                 success: this.clear_errors_then(function() {
                     return alert(fullSuccessMessage);  // eslint-disable-line no-alert
                 }),
-                error: statusAjaxError(function() {
+                error: statusAjaxError(function(response) {
+                    if (response.responseText === 'Specified module does not support rescoring.') {
+                        return that.$request_err_grade.text(rescoreUnsupportedMessage);
```

Comment:
I'm going to look at writing a bokchoy test for this functionality, too, so that something will break if anyone changes the string underlying the current logic.